CONTRIBUTING.md

```diff
@@ -35,9 +35,9 @@ If you want to help resolve an issue by making some changes that are larger than
 - Ensure you have nodejs
 - Run the following commands inside the cloned repository:
   - `pip install -e .[dev]` - This will install the Python package in development
-    mode.
+    mode. (note: if you get errors related to pyqt5 or pyqtwebengine version, run `pip install pyqt5==5.9` and `pip install pyqtwebengine` respectively)
```

Comment:
I don't think this is related.

Reply:
These are the errors that I ran into while installing, and I found many instances of other users online with the same issue. I guess it might be too specific to include since there are many requirements that are checked.

Reply:
we don't have any dependencies on these, so your errors' root cause is in one of our dependencies, not on us.

Reply:
That makes sense. How do I update my pull request to reflect those changes?

Reply:
just commit the changes in your local repo on your local branch `patch-1` then `git push origin patch-1` and it will auto update the PR